### PR TITLE
Update the reference for text_to_video.md

### PR DESCRIPTION
**Pull request by JacobYuan7**
The original reference (VideoFusion) might be misleading. VideoFusion is not open-sourced. I am the co-first author of ModelScopeT2V. I change the referred paper to the right one.

# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/en/api/pipelines/text_to_video.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ specific language governing permissions and limitations under the License.
 
 # Text-to-video
```
Comment (stevhliu) [inline docs/source/en/api/pipelines/text_to_video.md:19]: Would you mind also updating the title in the [`toctree`](https://github.com/huggingface/diffusers/blob/main/docs/source/en/_toctree.yml) to align with the title here? I also think specifying it as _ModelScope_ Text-to-Video will make it clearer this isn't a generic model and it is more precise. 😄 

```suggestion
# ModelScope Text-to-Video
```

File: docs/source/en/api/pipelines/text_to_video.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ specific language governing permissions and limitations under the License.
...
 
-[VideoFusion: Decomposed Diffusion Models for High-Quality Video Generation](https://huggingface.co/papers/2303.08320) is by Zhengxiong Luo, Dayou Chen, Yingya Zhang, Yan Huang, Liang Wang, Yujun Shen, Deli Zhao, Jingren Zhou, Tieniu Tan.
+[ModelScope Text-to-Video Technical Report](https://arxiv.org/abs/2308.06571) is by Jiuniu Wang*, Hangjie Yuan*, Dayou Chen*, Yingya Zhang*, Xiang Wang, Shiwei Zhang.
```
Comment (stevhliu) [inline docs/source/en/api/pipelines/text_to_video.md:21]:
```suggestion
[ModelScope Text-to-Video Technical Report](https://arxiv.org/abs/2308.06571) is by Jiuniu Wang, Hangjie Yuan, Dayou Chen, Yingya Zhang, Xiang Wang, Shiwei Zhang.
```

**Review by stevhliu — APPROVED** [review]
Ah, very sorry for the incorrect reference! 😅

**Comment by JacobYuan7** [comment]
@stevhliu Many thanks for the prompt response! But not sure whether this edit has been merged to the main branch.

File: docs/source/en/api/pipelines/text_to_video.md (hunk 2 above)
Comment (stevhliu) [reply docs/source/en/api/pipelines/text_to_video.md:21]: Would you mind removing the `*`? I don't think we need to italicize every other author :)

Comment (JacobYuan7) [reply docs/source/en/api/pipelines/text_to_video.md:21]: @stevhliu  I add them to denote equal contribution. But, you can remove it if it is more appropriate.

**Review by JacobYuan7 — COMMENTED** [review]
I am good with this edit. Thanks.

**Comment by patrickvonplaten** [comment]
Thanks for the correction @JacobYuan7

**Comment by JacobYuan7** [comment]
@patrickvonplaten Pleasure.

**Comment by JacobYuan7** [comment]
@stevhliu May I know when this change will be shown in the default Stable version (V0.23.0) of [Diffusers](https://huggingface.co/docs/diffusers/v0.23.1/en/api/pipelines/text_to_video)? In this version, this reference information is still the old one.

**Comment by stevhliu** [comment]
Hi, this change is currently live on the [main](https://huggingface.co/docs/diffusers/main/en/api/pipelines/text_to_video) version and it'll be shown in the stable version when we do the next release. 

cc @patrickvonplaten do you know when this will be?

**Pull request merged**